config/configauth/configauth.go

```diff
@@ -22,39 +22,39 @@ import (
 )
 
 var (
-	errAuthenticatorNotFound  = errors.New("authenticator not found")
-	errNotClientAuthenticator = errors.New("requested authenticator is not a client authenticator")
-	errNotServerAuthenticator = errors.New("requested authenticator is not a server authenticator")
+	errAuthenticatorNotFound = errors.New("authenticator not found")
+	errNotClient             = errors.New("requested authenticator is not a client authenticator")
+	errNotServer             = errors.New("requested authenticator is not a server authenticator")
 )
 
-// Authentication defines the auth settings for the receiver.
-type Authentication struct {
+// Settings defines the auth settings for the receiver.
+type Settings struct {
```

Comment:
For the moment we need to keep the old name, will rename in a different PR with deprecation.